source/features/github-bugs.css

```diff
@@ -16,3 +16,13 @@
 	border-bottom-left-radius: 5px;
 	border-bottom-right-radius: 5px;
 }
+
+/* Style backticked code in links #4817 */
+.js-commits-list-item code, /* `isCommitList` commit message */
+code .markdown-title[data-test-selector='commit-tease-commit-message'] /* `isRepoTree` commit message */ {
```

Comment:
The selector might be a bit complex, but this should target `code` too, which works as suggested in https://github.com/sindresorhus/refined-github/pull/4817#discussion_r716028339
<img width="774" alt="Screen Shot 3" src="https://user-images.githubusercontent.com/1402241/134794995-28abc1c2-741e-4772-be95-36ede89a4449.png">

This works for example:

```suggestion
.Box-header .commit-author + span code /* `isRepoTree` commit message */ {
```